core/domain/exp_jobs_one_off.py

```diff
@@ -34,8 +36,18 @@
 from core.platform import models
 import feconf
 import python_utils
+import schema_utils
 import utils
 
+CURR_DIR = os.path.abspath(os.getcwd())
```

Comment:
This is done so that the path to the third-party lib can be resolved. Also, the linter disables are necessary because I couldn't add the path insert code above the imports, that would also cause the linter checks to fail.

Reply:
The other libs also require such explicit addition of paths. The lib you've mentioned has done this in multiple places that it's being used in. Since this is the only place pylatexenc is being used in, the path to it has been added here. Also, this sort of thing is being done in other places as well, for instance [here](https://github.com/oppia/oppia/blob/640cf5aff24da4df8e549f4d79b9b704ba74f213/scripts/linters/js_ts_linter.py#L35), the path to the esprima library has been explicitly added just the way I've added it here which is why I thought this was the norm.

Reply:
The lib I mentioned appears to be used in exactly one place (core/controllers/voice_artist.py) and, as far as I can tell, does not require any addition of paths or usage of linter disables. Could you please clarify what you are looking at?

Stuff in the scripts/ folder has different rules because it is run using cmdline, so don't take that as a basis for what you should do with third-party libs in the main codebase.

Reply:
I've fixed this now.

Reply:
Thanks. Could you please explain the earlier sentence about "The lib you've mentioned has done this in multiple places that it's being used in." -- what happened there?

Reply:
Ah, I saw the path being inserted in multiple files like `appengine_config.py`, `gae_suite.py`, `run_backend_tests.py`, and `pre_commit_linter.py`. I thought if I were to just add the path in this file, I wouldn't have to add it to multiple places like that. But I suppose that is more consistent, which is why I've added it to those files instead now.